.pre-commit-config.yaml

```diff
@@ -41,7 +41,7 @@ repos:
     rev: v4.0.0-alpha.8
     hooks:
       - id: prettier
-        types_or: [jsx, javascript, mdx, markdown]
+        types_or: [jsx, javascript]
```

Comment:
Not sure why this was removed but we can revisit in the future.

Reply:
prettier was reformatting the HTML tables embedded in the Markdown, adding significant whitespace that rendered the table weirdly. Ultimately, this is an issue that can be resolved by updating the CSS, but this was faster. Change is here: [`3f5fe0c` (#198)](https://github.com/conda/conda-dot-org/pull/198/commits/3f5fe0c8f946928469c95979ec5720190e4d0895)